internal/slicer/slicer_test.go

```diff
@@ -487,6 +487,35 @@ var slicerTests = []slicerTest{{
 		`,
 	},
 	error: `slice base-files_myslice: content is not a file: /x/y`,
+}, {
+	summary: "Non-default archive",
+	slices:  []setup.SliceKey{{"base-files", "myslice"}},
+	release: map[string]string{
+		"chisel.yaml": `
+			format: chisel-v1
+			archives:
+				foo:
+					version: 22.04
+					components: [main, universe]
+					default: true
+				bar:
+					version: 22.04
+					components: [main]
+		`,
+		"slices/mydir/base-files.yaml": `
+			package: base-files
+			archive: bar
+			slices:
+				myslice:
+					contents:
+						/usr/bin/hello:
+		`,
+	},
+	result: map[string]string{
+		"/usr/":          "dir 0755",
+		"/usr/bin/":      "dir 0755",
+		"/usr/bin/hello": "file 0775 eaf29575",
```

Comment:
Out of curiosity, how did you know the checksum of the `/usr/bin/hello` here?

Reply:
It is the same file from the embedded `base-files` package, so I've just copied it from the `result` maps from earlier test cases.